script/create_selenium_db.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+MB_SERVER_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/../" && pwd)
+cd "$MB_SERVER_ROOT"
+
+./script/create_test_db.sh SELENIUM
+
+./admin/psql SELENIUM < ./t/sql/selenium.sql
+
+if [[ $# -gt 0 ]]; then
+    EXTRA_SQL="$1"
+    if [[ -f "$EXTRA_SQL" ]]; then
+        ./admin/psql SELENIUM < "$EXTRA_SQL"
+    fi
+fi
+
+DROP_SQL=$(cat <<'SQL'
+\set ON_ERROR_STOP 1
+DROP EXTENSION IF EXISTS amqp CASCADE;
+DROP SCHEMA IF EXISTS artwork_indexer CASCADE;
+SQL
+)
+echo "$DROP_SQL" | ./admin/psql --system SELENIUM
+
+if [ -z "$SIR_DIR" ]; then
+    SIR_DIR="$MB_SERVER_ROOT"/../sir
+fi
```

Comment:
Tip for setting a variable to a default value:

```suggestion
SIR_DIR="${SIR_DIR:=$MB_SERVER_ROOT/../sir}"
```